draft/protos/ydb_object_storage.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+option cc_enable_arenas = true;
+
+package Ydb.ObjectStorage;
+option java_package = "com.yandex.ydb.object_storage";
```

Comment:
To compare with other [services](https://github.com/ydb-platform/ydb-api-protos/blob/master/protos/ydb_query.proto#L6) it will be better to use `tech.ydb.proto.object_storage`